backend/controller/controller.js

```diff
@@ -92,6 +92,8 @@ export const login = async (req, res, next) => {
 export const logout = (req, res) => {
   //this will attempt to destroy session and invalidate the token, on the front end we will also remove the token from cookie to make sure
   try {
+
+    console.log(`Log Out route: current user id ${req.user.user_id}`);
```

Comment:
This is a good print statement for clarity regarding how to access the logged in user's id.